app/src/main/java/org/mozilla/fenix/compose/MessageCard.kt

```diff
@@ -134,13 +121,83 @@ fun MessageCard(
 
                 PrimaryButton(
                     text = buttonText,
+                    textColor = messageColors.buttonTextColor,
+                    backgroundColor = messageColors.buttonColor,
                     onClick = onClick,
                 )
             }
         }
     }
 }
 
+/**
+ * IconButton within a MessageCard.
+ *
+ * @param messageColors The color set defined by [MessageCardColors] used to style the button.
+ * @param onCloseButtonClick Invoked when user clicks on close button to remove message.
+ */
+@Composable
+private fun MessageCardIconButton(
+    messageColors: MessageCardColors = MessageCardColors.buildMessageCardColors(),
```

Comment:
Let's stick to passing only the `tint` here since it doesn't need to know about any of the other colors.